src/providers/token-fee-fetcher.ts

```diff
@@ -27,31 +27,34 @@ type Address = string;
 export type TokenFeeResult = {
   buyFeeBps?: BigNumber;
   sellFeeBps?: BigNumber;
+  feeTakenOnTransfer?: boolean;
```

Comment:
you'll have to update GQL token fee fetcher to retrieve and expose those fields as well
see [here](https://github.com/Uniswap/routing-api/blob/19b8e7a9e591718233c056eb18a8abb211c61023/lib/graphql/graphql-token-fee-fetcher.ts#L49) and [here](https://github.com/Uniswap/routing-api/blob/19b8e7a9e591718233c056eb18a8abb211c61023/lib/graphql/graphql-queries.ts#L2)